evap/evaluation/management/commands/scss.py

```diff
@@ -23,6 +28,9 @@ def handle(self, *args, **options):
         if options['watch']:
             command += ['--watch', '--poll']
 
+        if options['release']:
```

Comment:
maybe call this "production"? i think that's how it's usually called in the javascript world (e.g. in webpack or rollup), "release" sounds more like compiled languages